cardano-cli/src/Cardano/CLI/EraBased/Commands/Governance/Actions.hs

```diff
@@ -54,8 +54,8 @@ data NewCommitteeCmd
     , ebReturnAddress :: AnyStakeIdentifier
     , ebProposalUrl :: ProposalUrl
     , ebProposalHashSource :: ProposalHashSource
-    , ebOldCommittee :: [AnyStakeIdentifier]
-    , ebNewCommittee :: [(AnyStakeIdentifier, EpochNo)]
+    , ebOldCommittee :: [VerificationKeyOrHashOrFile CommitteeColdKey] -- jky
```

Comment:
what's with the comment?

Reply:
It's basically John's version of "John Ky was here :call_me_hand: "

Reply:
Fixed!